spec/requests/omniauth_callbacks_controller_spec.rb

```diff
@@ -22,7 +22,7 @@ def set_mock_auth(trn)
         get "/claim/auth/tid/callback"
```

Comment:
Can't we use the proper rails path instead of URL?

Reply:
Sure, let me update it.